test/parallel/test-readline-keys.js

```diff
@@ -44,6 +44,39 @@ function addTest(sequences, expectedKeys) {
   assert.deepStrictEqual(keys, expectedKeys);
 }
 
+const addKeyIntervalTest = (sequences, expectedKeys, interval = 150,
+                            assertDelay = 150) =>
```

Comment:
@Fishrock123 , `(next)` is there to capture next test case and the following thunk `()` is to delay the actual call. Rest are belong to the `addKeyIntervalTest`. Comment helps ?